test/unittests/test_dd_synthesis_dc.cpp

```diff
@@ -38,35 +39,20 @@ INSTANTIATE_TEST_SUITE_P(TestDDSynth, TestDDSynthDc,
                                  "rd32_19",
                                  "c17",
                                  "con1",
-                                 "root",
```

Comment:
What's the purpose of removing these tests?

Reply:
Nothing major, the decoder ends up adding significant number of gates to the corresponding qc. As a result, the simulation time would also increase for these benchmarks. That is why I removed.

Reply:
Makes sense. At some point it would probably make sense to port the `simple_simulation` routine from the SyReC project to `qc::QuantumComputation`s. 
Even the simulation of a 64bit reversible circuit is trivial and shouldn't take long.
Actually, it shouldn't really take long with our DDs as well. In the end, the state representation will always consist of `n` nodes for an `n`-bit reversible function.